src/single-spa-vue.js

```diff
@@ -75,6 +75,10 @@ function mount(opts, mountedInstances, props) {
         }
       } else {
         domEl = appOptions.el;
+        if (!domEl.id) {
+          domEl.id = `single-spa-application:${props.name}`;
```

Comment:
This is behavior already implemented in `single-spa-react` yes?

Reply:
Yes, but this is single-spa-vue